packages/nextjs/contracts/deployedContracts.ts

```diff
@@ -4,6 +4,152 @@
  */
 import { GenericContractsDeclaration } from "~~/utils/scaffold-eth/contract";
 
-const deployedContracts = {} as const;
+const deployedContracts = {
```

Comment:
why do we want to add this contract?

Reply:
Ah I did not intend to include this, this must have been automatically generated by scaffold-eth ... feel free to remove / exclude from the PR, it shouldn't be part of this issue.

Reply:
Ah I did not intend to include this, this must have been automatically generated by scaffold-eth ... feel free to remove / exclude from the PR, it shouldn't be part of this issue.